notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,261 @@
+      "Requirement already satisfied: azure-nspkg>=3.0.0 in /data/anaconda/envs/py36/lib/python3.6/site-packages (from azure-mgmt-nspkg>=2.0.0->azure-mgmt-keyvault>=0.40.0->azureml-core==1.0.21.*->azureml-contrib-notebook>=1.0.21.2) (3.0.2)\r\n",
+      "Requirement already satisfied: pycparser in /data/anaconda/envs/py36/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.7->cryptography!=1.9,!=2.0.*,!=2.1.*,!=2.2.*->azureml-core==1.0.21.*->azureml-contrib-notebook>=1.0.21.2) (2.19)\r\n",
+      "Requirement already satisfied: oauthlib>=0.6.2 in /data/anaconda/envs/py36/lib/python3.6/site-packages (from requests-oauthlib>=0.5.0->msrest>=0.5.1->azureml-core==1.0.21.*->azureml-contrib-notebook>=1.0.21.2) (2.1.0)\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install \"azureml-contrib-notebook>=1.0.21.2\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%run ../../scripts/generate_conda_file.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up AzureML workspace\n",
+    "Please refer to [configuration.ipynb](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb) to set up Azure account, subscription, workspace and compute target. Below code assumes config.json file is in place."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from azureml.core import Workspace\n",
+    "from os import path, makedirs\n",
+    "import shutil\n",
+    "from azureml.core import Experiment\n",
+    "from azureml.core.runconfig import RunConfiguration\n",
+    "from azureml.contrib.notebook.notebook_run_config import NotebookRunConfig, PapermillExecutionHandler\n",
+    "from azureml.core.runconfig import DEFAULT_CPU_IMAGE\n",
+    "from azureml.core.conda_dependencies import CondaDependencies\n",
+    "\n",
+    "subscription_id = os.getenv(\"SUBSCRIPTION_ID\", default=\"<my-subscription-id>\")\n",
+    "resource_group = os.getenv(\"RESOURCE_GROUP\", default=\"<my-resource-group>\")\n",
+    "workspace_name = os.getenv(\"WORKSPACE_NAME\", default=\"<my-workspace-name>\")\n",
+    "workspace_region = os.getenv(\"WORKSPACE_REGION\", default=\"eastus2\")\n",
+    "\n",
+    "ws = Workspace(subscription_id = subscription_id, resource_group = resource_group, workspace_name = workspace_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example uses run-based AzureML Compute. Alternatively, you can use persistent compute target (cpu or gpu cluster) or remote virtual machines. For more details, see [How to set up training targets](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets). The VM size (`STANDARD_D2_V2`) used here costs $0.114/hour."
```

Comment:
Why did you choose to use run-based compute here?

Reply:
Because this one takes minimum setup and cleans up automatically. I'm hoping if user somehow want persistent compute, they can read the doc and it's not too much change from this one.

Reply:
We can also maybe link to the other sar notebook for an example of how to use persistent compute as well.

Reply:
I think I should put a link to the sar notebook here because they're both getting-started aml examples. But user won't be able to find how to use persistent compute there. 
If that's what user would be interested in, I can put a commented block to show how to use persistent compute.

Just to compare, cpucluster config looks like this:

> from azureml.core.runconfig import DEFAULT_CPU_IMAGE
> from azureml.core.conda_dependencies import CondaDependencies
> run_config = RunConfiguration()
> run_config.target = "cpucluster"
> run_config.environment.docker.enabled = True
> run_config.environment.docker.base_image = DEFAULT_CPU_IMAGE
> run_config.environment.python.user_managed_dependencies = False
> run_config.auto_prepare_environment = True

This is run-based compute config (as already in this notebook):

> run_config_aml = RunConfiguration()
> run_config_aml.target = "amlcompute" # possible to use different compute targets
> run_config_aml.amlcompute.vm_size = 'STANDARD_D2_V2'
> run_config_aml.environment.docker.enabled = True
> run_config_aml.environment.docker.base_image = DEFAULT_CPU_IMAGE
> run_config_aml.environment.python.user_managed_dependencies = False
> run_config_aml.auto_prepare_environment = True

Reply:
What do you mean?The SAR on azureml notebook is using persistent compute?
![image](https://user-images.githubusercontent.com/8547756/55514816-12cb1580-5637-11e9-8dd3-97d3d1e60663.png)

Reply:
I mean in addition to create persistent compute, as shown in the other notebook, user need to replace run_config.target and remove run_config_aml.amlcompute.vm_size.